data/strings/types_strings.txt

```diff
@@ -25947,7 +25948,7 @@
     bg = Хранителни стоки
     cs = Potraviny
     da = Købmand
-    de = Lebensmittelgeschäft
+    de = Unverderbliche Lebensmittel Geschäft
```

Comment:
"Unverderbliche Lebensmittel Geschäft" doesn't seem correct.
I think "Lebensmittelgeschäft" is fine.

Reply:
"Lebensmittelgeschäft" doesn't match the OSM definition of [shop=grocery](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dgrocery). But "Haltbare Lebensmittel" is a more concise name.

(Using the imprecise "Lebensmittelgeschäft" is a problem when adding places in the editor, but maybe we should just remove this niche category there (can_add=no) anyway.)

Reply:
I still prefer "Haltbare Lebensmittel" because not all long lasting good are packaged in cans.

Reply:
Wegen _Konservendose_? Konservieren geht auch mit Einmachgläsern oder pökeln, trocknen etc.

Reply:
Oh, stimmt. Dann können wir "Lebensmittelkonserven" nehmen.